ManageIQ/manageiq
# Add Gnocchi service for Openstack metrics

Gnocchi provides C&U functionality since it was moved form Ceilometer API. We need to add a support for the service. This implementation should be able handle both Ceilometer and Gnocchi based metric capture.

Links
----------------

* https://bugzilla.redhat.com/show_bug.cgi?id=1387310
* Required manageiq-gems-pending change https://github.com/ManageIQ/manageiq-gems-pending/pull/9
* downstream PR for backport https://github.com/ManageIQ/manageiq/pull/12547

Steps for Testing/QA
-------------------------------

* setup OSP10 and add it as Cloud provider
* enable metrics collection
* provision a VM
* invoke metrics capture or wait until it gets captured (up to 1 day)
* open Monitoring - Utilization on the VM detail page and you should see charts with meaningful data
